src/main/java/org/opentripplanner/transit/raptor/api/response/StopArrivals.java

```diff
@@ -10,12 +10,6 @@ public interface StopArrivals {
    */
   boolean reached(int stopIndex);
 
-  /**
-   * The earliest arrival time at the given stop. If the stop is not reached, the behavior is
-   * undefined; It may return an arbitrary value or throw an exception.
-   */
-  int bestArrivalTime(int stopIndex);
-
```

Comment:
These methods are useful when analysing and comparing results. I would like to make a tool for this - so I want to keep the methods in the API.

Reply:
```
 /**
   * The earliest arrival time at the given stop. If the stop is not reached, the behavior is
   * undefined; It may return an arbitrary value or throw an exception.
   * <p>
   * This is currently unused in OTP, but useful for analyses/comparing the results.
   */
  @SuppressWarnings("unused")
  int bestArrivalTime(int stopIndex);
```